Library/Homebrew/test/livecheck_spec.rb

```diff
@@ -106,6 +106,26 @@
     end
   end
 
+  describe "#version" do
+    it "returns nil if not set" do
+      expect(livecheckable.version).to be nil
+    end
+
+    it "returns value if set" do
+      livecheckable.version("foo")
+      expect(livecheckable.version).to eq("foo")
```

Comment:
```suggestion
      livecheckable.version("1.2.3")
      expect(livecheckable.version).to eq("1.2.3")
```

It wouldn't hurt to use a value that's a little closer to something that we would expect to see in practice, to make the test a bit more understandable.